pkg/webhook/owner_reference/patching.go

```diff
@@ -43,15 +43,15 @@ func Webhook(handler capsulewebhook.Handler) capsulewebhook.Webhook {
 	return &webhook{handler: handler}
 }
 
-func (w webhook) GetHandler() capsulewebhook.Handler {
-	return &handler{}
+func (w *webhook) GetHandler() capsulewebhook.Handler {
+	return w.handler
```

Comment:
@MaxFedotov this was the cause of the issue reported on #85: we weren't returning the passed handler but rather the old implementation.